content/v2/pedagogy/roles_data8.md

```diff
@@ -0,0 +1,126 @@
+## Roles and Responsibilities
+Each semester, around 45 teaching assistants (TAs), 45 tutors, and 150 academic interns (AIs) are hired to be part of Data 8 course staff. There are differing levels of hourly commitments, roles, and responsibilities both by and within roles depending on the staff member’s experience and ability to contribute to course staff.
+
+### Academic Intern (AIs)
+
+##### Lab assisting (2 hrs/week)
+- 10 out of the 13 weeks (20 hours) → required for P
```

Comment:
Clarify what the P means.

Reply:
Oh, it's for "Pass". Should I push the corrections and submit a new PR?

Reply:
Yeah, push corrections to this PR, no need to submit a new one.